src/site/content/en/blog/local-fonts/index.md

```diff
@@ -324,12 +293,6 @@ The Chrome team has designed and implemented the Local Font Access API using the
 defined in [Controlling Access to Powerful Web Platform Features][powerful-apis], including user
 control, transparency, and ergonomics.
 
-### User control
-
-Access to a user's fonts is fully under their control and will not be allowed unless the
-`"local-fonts"` permission, as listed in the
-[permission registry](https://w3c.github.io/permissions/#permission-registry), is granted.
-
```

Comment:
Turns out, `"local-fonts"` is not actually registered, see https://w3c.github.io/permissions-registry/#dfn-table-of-standardized-permissions-of-the-web-platform. So it seems better to not list it.